src/lmic/config.h

```diff
@@ -69,7 +69,21 @@
 // communicating with the radio.
 // The standard range is 125kHz-8MHz, but some boards can go faster.
 #ifndef LMIC_SPI_FREQ
-#define LMIC_SPI_FREQ 1E6
+# ifdef ARDUINO_CATENA_4550
+#  define LMIC_SPI_FREQ 4000000	/* 4MHz */
+# else
+#  define LMIC_SPI_FREQ 1E6
+# endif
+#endif
+
+// Change the TX/RX antenna polarity
+// By default, TX is 1 and RX is 0.
+#ifdef ARDUINO_CATENA_4550
+# define LMIC_ANTENNA_SWITCH_RX 1
+# define LMIC_ANTENNA_SWITCH_TX 0
```

Comment:
I think it would be better to add a polarity control to the wiring table. That would mean minor changes in the HAL but would let the same code support more configurations.